### PR TITLE
Update Google Cloud docs with Docker recipe

### LINKED ISSUE #2881: Should we remove the Docker examples from GCP deploy guide?
We've written and rewritten the GCP deploy guide a couple of times, before recipes were a thing.

Now that we've also added Docker as a recipe, should we remove the Docker examples in the GCP guide and refer to the recipe to remove duplicate content? Or keep it the way as it is to offer a full and complete guide for Google Cloud deploys?

I think this should be discussed!

### PR DESCRIPTION
**Pull request by ElianCodes**
#### What kind of changes does this PR include?

- New or updated content

#### Description

Updates the Google Cloud documentation to refer to [the Docker Recipe](<https://docs.astro.build/en/recipes/docker/>) instead of having double content.

- Closes #2881

### CODE REVIEW

**Comment by ElianCodes** [comment]
Page that this PR changes:

- Original: <https://docs.astro.build/en/guides/deploy/google-cloud/>
- Changed: <https://deploy-preview-2935--astro-docs-2.netlify.app/en/guides/deploy/google-cloud/>

File: src/content/docs/en/guides/deploy/google-cloud.mdx [hunk 1]
```diff
@@ -35,67 +35,10 @@ Cloud Run is a serverless platform that allows you to run a container without ha
...
-These are just examples of Dockerfiles. You can customize them to your needs. For example, you could use another image, like `node:lts-alpine`.
-
-```docker title="Dockerfile" del={1} add={2}
-FROM node:lts AS runtime
-FROM node:lts-alpine AS runtime
-```
-:::
-
-#### Build the Container
+Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](</en/recipes/docker/#creating-a-dockerfile>).
```
Comment (Yan-Thomas) [inline src/content/docs/en/guides/deploy/google-cloud.mdx:41]: I'm wondering why there are arrows in this link. Just to be sure we don't fall into some very specific error in the future, I'd remove this.
```suggestion
Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](/en/recipes/docker/#creating-a-dockerfile).
```

**Review by Yan-Thomas — COMMENTED** [review]
Nice @ElianCodes, thanks for this! I added a small suggestion, after addressing it LGTM 🙌

File: src/content/docs/en/guides/deploy/google-cloud.mdx (hunk 1 above)
Comment (ElianCodes) [reply src/content/docs/en/guides/deploy/google-cloud.mdx:41]: 😱 I always do this! seems like it's not needed at all 😅 It does work tho!

**Comment by ElianCodes** [comment]
@Yan-Thomas I also fixed my previous mistakes in PR #2936

File: src/content/docs/en/guides/deploy/google-cloud.mdx [hunk 2]
```diff
@@ -35,67 +35,10 @@ Cloud Run is a serverless platform that allows you to run a container without ha
...
-These are just examples of Dockerfiles. You can customize them to your needs. For example, you could use another image, like `node:lts-alpine`.
-
-```docker title="Dockerfile" del={1} add={2}
-FROM node:lts AS runtime
-FROM node:lts-alpine AS runtime
-```
-:::
-
-#### Build the Container
+Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](/en/recipes/docker/#creating-a-dockerfile).
```
Comment (sarah11918) [inline src/content/docs/en/guides/deploy/google-cloud.mdx:41]:
```suggestion
Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. Find more information about [how to use Docker with Astro](/en/recipes/docker/#creating-a-dockerfile) in our recipe section.
```
I love a PR that deletes so many lines!  :partying_face: I'm just tightening up the call to action, then merging! Bye bye, all these lines!

**Pull request merged**